crates/blockchain/Cargo.toml

```diff
@@ -7,13 +7,27 @@ edition = "2021"
 
 [dependencies]
 thiserror.workspace = true
-ethereum_rust-core.workspace = true
 ethereum_rust-rlp.workspace = true
-ethereum_rust-storage.workspace = true
-ethereum_rust-vm.workspace = true
 sha3.workspace = true
 tracing.workspace = true
 
+# FIXME: It is necessary not to re-use the workspace definitions.
+ethereum_rust-core = { path = "../common", default-features = false }
+ethereum_rust-storage = { path = "../storage/store", default-features = false }
+ethereum_rust-vm = { path = "../vm", default-features = false }
```

Comment:
Why can't we set `default-features = false` in the workspace dependencies?

Reply:
Can't we just import the crates with the "libmdbx"  feature in `ethereum_rust`'s Cargp.toml?